src/segments/client.py

```diff
@@ -664,6 +664,7 @@ def clone_dataset(
         new_task_type: Optional[TaskType] = None,
         new_public: Optional[bool] = None,
         organization: Optional[str] = None,
+        clone_labels: Optional[bool] = None,
```

Comment:
Should default to `False`, not `None`. `None` is only to indicate "leave it as is".

Reply:
Makes sense. Changed ✅